package.json

```diff
@@ -6,17 +6,32 @@
   "scripts": {
     "lint": "eslint .",
     "test": "grunt test",
+	"mocha": "./node_modules/.bin/mocha --require @babel/register test/testHelper.js --recursive test/suite  --colors",
     "build": "grunt build"
   },
+  "dependencies": {
+	  "jquery": "~3.4.1",
+	  "@babel/polyfill": "^7.0.0"
+  },
   "devDependencies": {
     "eslint": "^5.12.0",
-    "eslint-config-wikimedia": "^0.5.0",
+    "eslint-config-wikimedia": "^0.8.1",
+	"eslint-loader": "1.1.0",
     "grunt": "^1.0.2",
+    "grunt-browserify": "^5.3.0",
     "grunt-concat-with-template": "^0.1.0",
     "grunt-contrib-concat": "^1.0.1",
     "grunt-contrib-qunit": "^2.0.0",
-    "grunt-contrib-watch": "^1.1.0",
     "grunt-eslint": "20.1.0",
-    "qunit": "^2.6.1"
+	"grunt-run": "~0.8.1",
+	"@babel/cli": "^7.1.5",
+    "@babel/core": "^7.1.6",
+    "@babel/node": "^7.0.0",
+    "@babel/preset-env": "^7.1.6",
+    "@babel/register": "^7.0.0",
+	"babelify": "~10.0.0",
+    "chai": "^3.4.1",
+    "mocha": "^2.3.4",
+	"jsdom": "~15.1.1"
```

Comment:
Inconsistent indentation in this file; some are tabs, other spaces

Reply:
Yeah, I need to clean up the package.json in general, I think. I was doing a bunch of testing with several packages, some of those aren't actually used in the final result. I'll go over that + the indentations.